[DOC] process execution data - examples of data (#1310)
process-analytics/bpmn-visualization-js: docs/users/overview.adoc

image::images/bpmn-diagram_navigation_C.2.0.gif[BPMN Diagram Navigation]

_Navigation with the https://github.com/bpmn-miwg/bpmn-miwg-test-suite/blob/cc75e467fd2b3009e67d4b24943591c66ce91a23/Reference/C.2.0.bpmn[migw-test-suite C.2.0] BPMN diagram_

> === Purpose

Reviewer: ```suggestion
=== Process data

```

> Process Analytics gives you the opportunity to visualize the Process Data on top of the BPMN diagram.

Reviewer: ```suggestion
`bpmn-visualization` gives you the opportunity to visualize the Process Data on top of the BPMN diagrams.
```

> Process Analytics gives you the opportunity to visualize the Process Data on top of the BPMN diagram.

Reviewer: In addition, I will add a link on `BPMN diagrams` that goes to the related paragraph of the page.

> The purpose of the library is to gain the visibility on what happens in your processes.

Reviewer: Translation tools propose 'The purpose of the library is to gain visibility into what is going on in your processes' but I am not sure of this sentence

Author: hehehe, we must ask Mickey to check it for us:)

for me both make sense.

Reviewer: Ok fine, let's keep it as it is now, we will request a review later when all doc update we are working on will be completed.write the whole documentation.

> Process Data consists of two major data sets:

Reviewer: alternate sentence
```suggestion
Process Data consists of two main data sets:
```

> * Execution Data - data that accompany the process instance (case) through the execution flow. All the events occurring along this workflow, contractual data and every other bits of information related to this particular instance from start to the end.

Reviewer: ```suggestion
* Execution Data - data that accompanies the process instance (case) through the execution flow. All the events occurring along this workflow, contractual data and every other bits of information related to this particular instance from start to the end.
```

* Derived Data - data that derives from the execution data: predictions, analysis, statistics. The other data that can be associated with the process instance can also be a part of this set.

The example of Process Data for different tasks can be as follows.

"user task":

> - time of the execution

Reviewer: ⚠️ for consistency with the syntax already used in this page, I would prefer we use the following syntax for the list (see https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/#nested-unordered-list)
```
* data related to list of activities done to complete this task:
** SAP data manipulations
```

- percentage of the time spent in this task in the scope of whole time to finish the process instance
- data related to list of activities done to complete this task:
    * SAP data manipulations
    * messaging system data

>     * excel entries (reporting)

Reviewer: ```suggestion
    * Microsoft Excel entries (reporting)
```

    * other proprietary systems data

"service task":

- time of the execution

> - data related to list of activities done to complete this task:
>     * RPA robot data

Reviewer: We need to describe more examples here
```suggestion
- data related to list of activities done to complete this task:
    * RPA robot data
    * external API call
    * ...
```
